packages/asc-ui/src/components/Header/HeaderTallTitle.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+// import HeaderTitleStyle from '../../styles/components/HeaderStyle/HeaderTitleStyle'
+import HeaderTallLogo from './HeaderTallLogo'
+import styled from '../../styled-components'
+
+type Props = {
+  title: string
+  homeLink: string
+}
+
+const HeaderTallTitleStyle = styled.div`
+  & > h1 {
+    display: flex;
+    height: 100%;
+    margin: 0;
+  }
+
+  & > h1 > span {
```

Comment:
Why not extend the styling for `HeaderTitleStyle`. Writing it like this takes out the advantage of `styled-components` and we could just use inline css instead

Reply:
I don't think I understand what you mean. Can you post an example?